packages/bot-web-ui/src/components/transactions.jsx

```diff
@@ -29,27 +54,13 @@ class Transactions extends React.PureComponent {
                     </span>
                 </div>
                 <div className='transactions__content'>
-                    <ThemedScrollbars autoHide style={{ height: 'var(--drawer-scroll-height)' }}>
-                        {elements.map((element, index) => {
-                            switch (element.type) {
-                                case transaction_elements.CONTRACT: {
-                                    const contract = element.data;
-                                    return <Transaction key={`${contract.reference_id}${index}`} contract={contract} />;
-                                }
-                                case transaction_elements.DIVIDER: {
-                                    const run_id = element.data;
-                                    return (
-                                        <div key={run_id} className='transactions__divider'>
-                                            <div className='transactions__divider-line' />
-                                        </div>
-                                    );
-                                }
-                                default: {
-                                    return null;
-                                }
-                            }
-                        })}
-                    </ThemedScrollbars>
+                    <List
```

Comment:
Can we wrap this in ThemedScrollbars?
Refer [react-virtualized-in-trader](https://github.com/balakrishna-binary/deriv-app/commit/ccb77a5a0b74bba0db6f5778f3edd341ebcf87b6)